docs/user-guide/how-to.md

```diff
@@ -123,6 +123,54 @@ jobs:
 Read more about [terraform-docs GitHub Action] and its configuration and
 examples.
 
+## Pre-commit Hook
+
+With [`pre-commit`], you can ensure your Terraform module documentation is kept
+up-to-date each time you make a commit.
+
+First, simply create or update a `.pre-commit-hooks.yaml`
```

Comment:
Is this a typo? I think this should be `.pre-commit-config.yaml` for consumers of this git-hook.

Reply:
Good catch! Thank you!